erpnext/erpnext_integrations/doctype/taxjar_settings/taxjar_settings.json

```diff
@@ -104,11 +102,25 @@
    "label": "Nexus",
    "options": "TaxJar Nexus",
    "read_only": 1
+  },
+  {
+   "fieldname": "current_company",
```

Comment:
Fieldname should be just company, why current_company?

Reply:
My bad, was confused with doc.company, but that was in sales invoice. Will fix it.